stdlib/tests/crypto/falcon.rs

```diff
@@ -172,7 +173,7 @@ fn test_falcon512_probabilistic_product_failure() {
     expect_exec_error!(
         test,
         ExecutionError::FailedAssertion {
-            clk: 17490,
+            clk: 31615,
```

Comment:
Note: I changed the hardcoded value, as I expect the increase in clock cycles needed to be related to the fact that we don't merge basic blocks anymore.

Reply:
Indeed, confirmed. I ran the test before and after the removal and `combine_basic_blocks()`, and the cycle count jumped 2x. Pretty massive!